include/freerdp/server/cliprdr.h

```diff
@@ -68,6 +68,12 @@ struct _cliprdr_server_context
 	void* handle;
 	void* custom;
 
+	/* clipboard capabilities - set by server */
```

Comment:
Exactly - the server first announces its capabilities (i.e. support for long format names) - if the client replies that it does not support that capability the server will revert to use use short format names in that example.